psydac/api/tests/test_postprocessing.py

```diff
@@ -55,31 +55,27 @@ def build_2_mapped_squares():
     D1     = mapping_1(A)
     D2     = mapping_2(B)
 
-    domain = D1.join(D2, name = 'domain',
-                bnd_minus = D1.get_boundary(axis=1, ext=1),
-                bnd_plus  = D2.get_boundary(axis=1, ext=-1))
-    return domain
+    domains = [D1, D2]
+    connectivity = [((0,1,1),(1,1,-1))]
+    return Domain.join(domains, connectivity, 'domain')
 
 
 def build_2_squares():
     A = Square('A',bounds1=(0.5, 1.), bounds2=(0, np.pi/2))
     B = Square('B',bounds1=(0.5, 1.), bounds2=(np.pi/2, np.pi))
 
-    domain = A.join(B, 'domain',
-                    bnd_minus=A.get_boundary(1, 1),
-                    bnd_plus=B.get_boundary(1, -1))
-    return domain
+    domains = [A, B]
+    connectivity = [((0,1,1),(1,1,-1))]
+    return Domain.join(domains, connectivity, 'domain')
```

Comment:
As above.

Reply:
done in [d66a68e](https://github.com/pyccel/psydac/pull/309/commits/d66a68ed7d030e41dc86823c805ed685b78d906e)